### ISSUE TITLE
match doesn't show up in long lines

### ISSUE THREAD

**Issue by dmd** [thomasschafer/scooter#4]
On the results page, if the match is on a long line, the line is still displayed from the beginning, rather than aligned to show the match.

**Comment by thomasschafer** [comment]
Good point, thanks for raising. I want to improve the diff view so that the exact characters being changed are highlighted, so I'll make this bug fix at the same time
